src/core/core.scale.js

```diff
@@ -1466,6 +1511,12 @@ export default class Scale extends Element {
       const item = items[i];
       const tickFont = item.font;
       const label = item.label;
+
+      if (item.backdrop) {
```

Comment:
Because this happens before label rotation, angled labels are not properly overlapped. Opening a PR for a fix soon.